packages/webview_flutter/webview_flutter/lib/webview_flutter.dart

```diff
@@ -6,6 +6,7 @@ library webview_flutter;
 
 export 'package:webview_flutter_platform_interface/webview_flutter_platform_interface.dart'
     show
+        HttpAuthRequest,
```

Comment:
This export should also be tested in `packages/webview_flutter/webview_flutter/test/webview_flutter_test.dart`.